library/general/automount-serviceaccount-token/template.yaml

```diff
@@ -3,6 +3,7 @@ kind: ConstraintTemplate
 metadata:
   name: k8spspautomountserviceaccounttokenpod
   annotations:
+    title: "Automount Service Account Token for Pod"
```

Comment:
Maybe we can have a more semantically (as opposed to usage-oriented) name for this field?

e.g. `prettyName` or similar.

Also, WRT annotations... should we be careful about using fully-qualified names to avoid collisions? And if we want to make these features standard, we may want to integrate these fields into the actual ConstraintTemplate schema.

Reply:
Could you please elaborate with a couple eg for me to get started?

Reply:
so for fully-qualified, an example would be `metadata.gatekeeper.sh/title`

For names, I'm imagining this string could live both as a title to a web page and as a one-line entry in a table of constraints that describes what it does at a high-level.

`mediumName` (vs `shortName`) or `longName`

`oneLiner`

`elevatorPitch` <- I don't like this one so much

`tldr`

WRT putting it in as a field... curious about your thoughts on that.

Reply:
Fully qualified makes sense. I have updated it to `metadata.gatekeeper.sh/title`